codegen/templates/manifests/crd.yamltmpl

```diff
@@ -0,0 +1,30 @@
+[[- range $crd := .Crds -]]
+[[- if and (and (lt (len $crd.Spec.Versions) 2) (string_contains (index $crd.Spec.Versions 0).Name "alpha")) (should_not_skip $crd.Spec.Group (index $crd.Spec.Versions 0).Name $.ShouldSkip) ]]
+{{- if has "[[ $crd.GetName ]]/[[ (index $crd.Spec.Versions 0).Name ]]" $.Values.enabledExperimentalApi }}
```

Comment:
this value should be injected from GM (the consumer) `$.Values.enabledExperimentalApi`

Reply:
You mean the name of the field?

Reply:
Do we need this to be this configurable? There's only one consumer for this (as far as I can tell), perhaps we could agree on a name that works in all contexts, as they are probably going to be very similar?

Reply:
If we do need this to be configurable, could we push this change until after 2.4 release?

Reply:
done.